ripple/src/main/webapp/posts.js

```diff
@@ -0,0 +1,116 @@
+/* Update function to fetch all comments for a particular post */
+function viewAllPostComments() {
+  alert("Fetch all comments for this post!");
+}
+
+/* Display an alert if user presses enter to comment on a post */
+function addComment(e) {
+  comment = document.getElementById("add-comment").value;
+  if (e.keyCode === 13) {
```

Comment:
There is another place checking for value being 13.
Can we make a global variable ENTER_KEY and use it.

Reply:
Fixed